HW1/task2.py

```diff
@@ -0,0 +1,17 @@
+# Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+print("Введите количество журавликов, сделанное всеми детьми")
+S = int(input())
+k = round(S/3/2)
```

Comment:
Наверное, проще использовать оператор целочисленного деления.
```suggestion
k = S // 3 // 2
```